week2/homework/src/index.js

```diff
@@ -1,3 +1,112 @@
 'use strict';
-
 // TODO: Write the homework code in this file
+
+const fs = require("fs");
+
+const help = [
+  "Run 'list' to show the current To-Do list, the item must be inside quotes ' '.",
+  "Run 'add' and then the Item you want to add to a To-Do list, the item must be inside quotes ' '.",
+  "Run 'remove' and then the Item then the item's Number to remove item from To-Do list, the item must be inside quotes ' '.",
+  "Run 'reset' to remove all To-Do items from the list, the item must be inside quotes ' '.",
+  "Run 'update' and then the item's Number you want to update in To-Do list then New Text, the item must be inside quotes ' '."
+];
```

Comment:
in an help file you should give very short descriptions
- list - show all the items
- add "todo item" - add "todo item" to the list
- update 2 "other todo item" - updates item at index 2 with "other todo item"
- remove 2 - removes item at index 2
- reset - clears the list